cloud

```diff
@@ -965,22 +986,25 @@ function execute_command() {
             --smtp-password-secret="$smtp_password_secret" \
             --database="$database"
         sections_run "$sections" scheduler_deploy \
-            "$project" "$prefix" "$trigger_topic"
+            "$project" "$prefix" "$load_queue_trigger_topic" \
+             "$pick_notifications_trigger_topic"
```

Comment:
Please move this line one character to the left, so it aligns with the previous one.